### PR TITLE
Run tests on local Artifactory

### PR DESCRIPTION
**Pull request by RobiNino**
- [x] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -45,21 +45,14 @@ jobs:
...
-          repository: RobiNino/jfrog-testing-infra #todo change to jfrog
-          path: jfrog-testing-infra
-          ref: rt-setup #todo remove
-
-      - name: Setup Artifactory
+      - name: Setup Artifactory # TODO change to jfrog
         run: |
-          cd jfrog-testing-infra
-          go run main.go
+          go install github.com/RobiNino/jfrog-testing-infra/local-rt-setup@latest
```
Comment (yahavi) [inline .github/workflows/tests.yml:51]: I think the latest is the default
```suggestion
          go install github.com/RobiNino/jfrog-testing-infra/local-rt-setup
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -78,8 +87,13 @@ jobs:
       - name: Run go vet
         run: go vet -v ./...
 
-      - name: Run tests
+      - name: Run Artifactory / Build tools tests
+        run: go test -v github.com/jfrog/jfrog-cli --timeout 0 --test.${{ matrix.suite }}=true --jfrog.url='http://localhost:8081/' --jfrog.user='admin' --jfrog.password='password' --ci.runId=${{ runner.os }}-${{ matrix.suite }}
```
Comment (yahavi) [inline .github/workflows/tests.yml:91]: Remove default values
```suggestion
        run: go test -v github.com/jfrog/jfrog-cli --timeout 0 --test.${{ matrix.suite }}=true --ci.runId=${{ runner.os }}-${{ matrix.suite }}
```

File: utils/tests/utils.go [hunk 3]
```diff
@@ -96,7 +97,8 @@ func init() {
 	DockerRepoDomain = flag.String("rt.dockerRepoDomain", "", "Docker repository domain")
 	DockerVirtualRepo = flag.String("rt.dockerVirtualRepo", "", "Docker virtual repo")
 	DockerRemoteRepo = flag.String("rt.dockerRemoteRepo", "", "Docker remote repo")
-	DockerLocalRepo = flag.String("rt.DockerLocalRepo", "", "Docker local repo")
+	DockerLocalRepo = flag.String("rt.dockerLocalRepo", "", "Docker local repo")
+	DockerPromoteLocalRepo = flag.String("rt.dockerPromoteLocalRepo", "", "Docker promote local repo")
```
Comment (yahavi) [inline utils/tests/utils.go:101]: Please document `rt.dockerPromoteLocalRepo` in the README
https://github.com/RobiNino/jfrog-cli/tree/tests-local-rt#docker-tests

**Review by yahavi — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (RobiNino) [reply .github/workflows/tests.yml:51]: Tag is required:
```
$ go install github.com/RobiNino/jfrog-testing-infra/go
no required module provides package github.com/RobiNino/jfrog-testing-infra/go; to add it:
	go get github.com/RobiNino/jfrog-testing-infra/go

File: README.md [hunk 4]
```diff
@@ -191,12 +191,13 @@ In addition to [general optional flags](#Usage) you _must_ use the following doc
...
-| `-rt.dockerRepoDomain`  | Artifactory Docker registry domain.         |
-| `-rt.dockerVirtualRepo` | Artifactory Docker virtual repository name. |
-| `-rt.dockerRemoteRepo`  | Artifactory Docker remote repository name.  |
-| `-rt.DockerLocalRepo`   | Artifactory Docker local repository name.   |
+| Flag                         | Description                                                    |
+|------------------------------|----------------------------------------------------------------|
+| `-rt.dockerRepoDomain`       | Artifactory Docker registry domain.                            |
+| `-rt.dockerVirtualRepo`      | Artifactory Docker virtual repository name.                    |
+| `-rt.dockerRemoteRepo`       | Artifactory Docker remote repository name.                     |
+| `-rt.DockerLocalRepo`        | Artifactory Docker local repository name.                      |
```
Comment (yahavi) [inline README.md:199]:
```suggestion
| `-rt.dockerLocalRepo`        | Artifactory Docker local repository name.                      |
```

**Review by yahavi — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -36,15 +36,24 @@ jobs:
...
       - name: Checkout code
         uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.17.x
+      - name: Setup Artifactory # TODO change to jfrog
```
Comment (eyalbe4) [inline .github/workflows/tests.yml:49]: I think this TODO comment isn't needed.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 5 above)
Comment (RobiNino) [reply .github/workflows/tests.yml:49]: It is for me, to stop using my fork

**Pull request merged**